README.md

```diff
@@ -167,9 +167,28 @@ As mentioned above, the Report Type is the type of report for which to generate
 | 15         | Net Worth Over Time |
 | 16         | Net Income Over Time |
 
-#### Date Filter
+### Financial Data Transactions
 
-As mentioned above, the Date Filter is the date window for which to generate your trend analysis.  The supplied value must be one of the following enum values:
+If you want to provide a more granular filtering of your financial data transactions, you can select from a variety of search filters that are sent to Mint. 
+
+| Parameter         | Data Type          | Description  |
+| ----------------  | ------------------ | -----------  |
+| date_filter       | DateFilter.Options | The date window for which to filter your transactions. |
+| start_date        | Optional[str]      | An optional beginning date to your transaction filtering. |
+| end_date          | Optional[str]      | An optional ending date to your transaction filtering. |
```

Comment:
Could we include the date format in the descriptions for start/end so it's easy to understand how to use?

Reply:
@mrooney you got it!